psetv2/blinder_role.go

```diff
@@ -495,7 +502,7 @@ func (b *BlinderRole) blindOutputs(
 		}
 
 		assetCommitment, err := b.blinderSvc.AssetCommitment(
-			output.outputAsset,
+			output.outputAsset[1:],
```

Comment:
would leave a comment here or wrap in a descriptive variable (ie. conf/unconf prefix)